src/compiler/diagnosticMessages.json

```diff
@@ -175,23 +175,23 @@
         "category": "Error",
         "code": 1057
     },
-    "Operand for 'await' does not have a valid callable 'then' member.": {
+    "Type used as operand to 'await' or the return type of an async function must not contain a callable 'then' member if it is not a promise.": {

```

Comment:
This double negative at the end is hard to read. Would it be accurate to reword as " ... may only contain a callable 'then' member if it is a promise." ?